deployment/README.md

```diff
@@ -0,0 +1,4 @@
+# Various deployemnt starategies of the app
+
+- [ ] Docker
```

Comment:
We do have an example how to start services with docker-compose. We could refers to `docker` folder here.
```suggestion
- [x] [Docker](../docker/)
```

Reply:
I know. It makes sense to mode this into the /deployment/docker folder.